ckanext/dcatapchharvest/profiles.py

```diff
@@ -38,6 +38,10 @@
 EMAIL_MAILTO_PREFIX = 'mailto:'
 ORGANIZATION_BASE_URL = 'https://opendata.swiss/organization/'
 
+DATE_FORMAT = "%Y-%m-%d"
+YEAR_MONTH_FORMAT = "%Y-%m"
+YEAR_FORMAT = "%Y"
```

Comment:
only because I very vaguely remember, that I added some constants similar to this in ckanext-switzerland, I first thought, that those kind of contants might be worth being read from the ini-file. But then, as these are pretty finicky, they need to be static as is. Even if that would make sense, I think it would rather be for the future. But maybe it is worth adding a section to the readme of this plugin so that it easier to understand how the validation works?
Not a blocker for this PR, but if a couple sentences can be done, that would be great :)